app/src/components/containers/initialDataContainer/initialDataContainer.jsx

```diff
@@ -44,6 +44,7 @@ export class InitialDataContainer extends Component {
   };
 
   componentDidMount() {
+    this.props.resetTokenAction();
```

Comment:
After success login and page refreshing we will reset token in `localStorage`.
It isn't expected (see the screenshot below).
![image](https://user-images.githubusercontent.com/31861805/54147967-62595f80-4444-11e9-87d5-e98106957650.png)

Reply:
fixed